Cargo.toml

```diff
@@ -4,6 +4,7 @@
 resolver = "2"
 members = [
   "core",
+  "core/examples/snapshot",
```

Comment:
This was the only way I found to get this example crate to run `build.rs`, which is required for generating the V8 Snapshot.  But I'd be happy to learn that there's a better way.